recipes/pertpy/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "pertpy" %}
+{% set version = "0.11.0" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pertpy-{{ version }}.tar.gz
+  sha256: 34edcd78b97a1f336ccd908c205f9c095bcb0247a901273f170cf73c1d580c40
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - rich
+    - scanpy
+    - mudata
+    - requests
+    - scikit-misc
+    - scvi-tools
+    - adjusttext
+    - decoupler-py
+    - ott-jax
+    - sparsecca
+    - openpyxl
+    - pubchempy
+    - pyarrow
+    - blitzgsea
+    - scikit-learn >=1.4
+    - fast-array-units
+    - lamin_utils
+    - arviz
+
+test:
+  imports:
+    - pertpy
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://pertpy.readthedocs.io
+  summary: Perturbation Analysis in the scverse ecosystem (incl. [ta]scCODA support).
```

```suggestion
  summary: Perturbation Analysis in the scverse ecosystem
```

Nit: A bit noisy so I'd just omit this detail.

---

Absolutely. Plus, it's a lie. tascCODA is actually not included. This is a remnant of an earlier attempt to package an older version.
Thank you for the catch!

I'll do this locally, squash and force-push once I want to re-run the checks.